lms/static/js/staff_debug_actions.js

```diff
@@ -67,7 +67,8 @@ var StaffDebug = (function(){
                 {text: text},
                 {interpolate: /\{(.+?)\}/g}
             )
```

Comment:
I ran JSHint on the files and changed all the syntactical errors that could be changed.